docs/doc/kbd.FAQ-15.html

```diff
@@ -188,7 +188,7 @@ <H2><A NAME="ss15.2">15.2 Sticky keys under <CODE>X</CODE></A>
 <P>XFree86 also supports Slow Keys, Repeat Keys, Bounce Keys and an
 audible bell. <CODE>xkbcomp</CODE> can be used to generate a <CODE>.xkm</CODE> file
 to enable these. The appropriate <CODE>xkbcomp</CODE> commands are listed in
-<CODE>/usr/X11R6/lib/X11/xkb/compat/accessx</CODE>.
+<CODE>/usr/{X11R6/lib,share}/X11/xkb/compat/accessx</CODE>.
```

Comment:
There is no need to create the illusion that this documentation is supported. Even in this hunk it is clear that we are talking about XFree86.